docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -501,6 +501,14 @@ of startup on its own.
 Tune the container's pids limit.
 This is equivalent to the Podman `--pids-limit` option.
 
+### `Pod=`
+
+Specify a pod to link the container to. This has the same format as the `--pod` option
+to `podman run`.
+
+As a special case, if the `name` of the pod ends with `.pod`, Quadlet will link the container
```

Comment:
```suggestion
As a special case, if the `name` of the pod ends with `.pod`, Quadlet will link the container.
```

Thanks!  That's what I should have paid attention to when I asked for dependencies.